### ISSUE TITLE
Module aw-server quit unexpectedly since Mar 29 [v0.9.2]

### ISSUE THREAD

**Issue by LENZFLIKER** [ActivityWatch/activitywatch#398]
**For us to be able to understand the issue you're having you need to follow this template.** Please make sure there isn't already an issue for the bug you've found, otherwise it'll just take more resources, which means less time for us to actually fix bugs. Thanks for reporting!

## Describe the bug
Worked all fine till March 28th but when I started my computer on 29th it showed these errors![](https://user-images.githubusercontent.com/17332970/77993332-90f2f280-7345-11ea-829b-f813d7cf0215.png). I tried uninstalling, wiping all data and reinstalling several times but the error kept showing.

## To Reproduce
1. Open Activity Watch.
2. Error.

## Expected behavior
Just work. Doesn't even start the localhost.

## Documentation
aw-server log files are blank.
[aw-qt_2020-03-31T06-06-59.log](https://github.com/ActivityWatch/activitywatch/files/4407122/aw-qt_2020-03-31T06-06-59.log)

## Environment
 - ActivityWatch version: [v0.9.2]
 - OS: [Windows 7]

## Additional context
I really need this for WFH stuff ;((

**Comment by johan-bjareholt** [comment]
If you are running Windows 7 I guess that the reason is that aw-server has not worked on windows 7 for a couple of years now

https://github.com/ActivityWatch/activitywatch/issues/91

The options are
- Fixing the bug (no one has taken an initiative yet to do so, no idea if it's easy or hard)
- Upgrading to Windows 10
- Switching to Linux or macOS

**Comment by ErikBjare** [comment]
I'm surprised you had it working at all, and also that aw-qt seems to run fine (which it hasn't for other Windows 7 users).

As Johan said it's unlikely this will ever be fixed, but out of curiosity: Could you start the aw-qt.exe/aw-server.exe from the console and show us the output?

**Comment by LENZFLIKER** [comment]
Yeah everything worked fine until that point. Anyways I ran aw-qt.exe and it didn't output anything. Below are few screenshots of the screen record I took from the output because cmd closes after showing the output and /WAIT or >__.txt didn't record anything.
![1](https://user-images.githubusercontent.com/17332970/78071928-e79b1380-73bb-11ea-9fd8-138aa2a6c156.png)
![2](https://user-images.githubusercontent.com/17332970/78071938-ea960400-73bb-11ea-9f71-3bb32dcabf89.png)
![3](https://user-images.githubusercontent.com/17332970/78071951-ee298b00-73bb-11ea-9114-96e90ab051f4.png)
